items.php

```diff
@@ -0,0 +1,394 @@
+<?php require_once 'engine/init.php'; include 'layout/overall/header.php';
+ 
+ if($config['items'] == true) {
+ 
+// Loading equipable items list
+$itemsCache = new Cache('engine/cache/items');
+if (user_logged_in() && is_admin($user_data)) {
+    if (isset($_GET['update'])) {
+        echo "<p><strong>Logged in as admin, loading engine/XML/items.xml file and updating cache.</strong></p>";
+        // ITEMS XML TO PHP ARRAY
+        $itemsXML = simplexml_load_file("engine/XML/items.xml");
+        if ($itemsXML !== false) {
+            $types = array();
+            $type_attr = array();
+            $groups = array();
+ 
+            // This empty array will eventually contain all items grouped by type and indexed by item type
+            $items = array();
+ 
+            // Loop through each XML item object
+            foreach ($itemsXML as $type => $item) {
+                // Get item types
+                if (!in_array($type, $types)) {
+                    $types[] = $type;
+                    $type_attr[$type] = array();
+                }
+                // Get item attributes
+                $attributes = array();
+                // Extract attribute values from the XML object and store it in a more manage friendly way $attributes
+                foreach ($item->attributes() as $aName => $aValue)
+                    $attributes["$aName"] = "$aValue";
+                // Remove unececsary attributes
+                if (isset($attributes['plural'])) unset($attributes['plural']);
+                //if (isset($attributes['id'])) unset($attributes['id']);
+                //if (isset($attributes['fromid'])) unset($attributes['fromid']);
+                //if (isset($attributes['toid'])) unset($attributes['toid']);
+                if (isset($attributes['editorsuffix'])) unset($attributes['editorsuffix']);
+                if (isset($attributes['article'])) unset($attributes['article']);
+                // Populate type attributes
+                foreach (array_keys($attributes) as $attr) {
+                    if (!in_array($attr, $type_attr[$type]))
+                        $type_attr[$type][] = $attr;
+                }
+ 
+                // Loop through every <attribute> object inside the <item> object
+				$item_attributes = array();
```

Comment:
I think its because there is a mix of tabs and 4 spaces used on this document. The comment is spaced with spaces, the variables are spaced with tabs. Github defaults tab to 8 spaces, while your notepad++? has a tab with of 4 spaces.
Tab should be used everywhere, so users can customize themselves wether they like to have 2, 4, 6 or 8 spaces as tab width.